### PR TITLE
Feat/export swc-Adds function to export SWC graph to a new SWC file

### PR DESCRIPTION
**Pull request by AdityaBITMESRA**
1. Adds function to export SWC graph to a new SWC file
2.Adds tests to verify if generated file and input file are similar

### CODE REVIEW

File: pyneuroml/swc/LoadSWC.py [hunk 1]
```diff
@@ -334,3 +334,22 @@ def load_swc(filename: str) -> SWCGraph:
             tree.add_node(node)
 
     return tree
+
+
+def export_swc(graph: SWCGraph, filename: str) -> None:
```
Comment (sanjayankur31) [inline pyneuroml/swc/LoadSWC.py:339]: I'd add this to the `SWCGraph` class as `export_to_swc_file`. It's still SWC graph related functionality.

File: tests/swc/.coverage
Comment (sanjayankur31) [inline tests/swc/.coverage:1]: Remove this file---it shouldn't be checked in. We should add `.coverage` to `.gitignore` I guess

File: tests/swc/test_LoadSWC.py [hunk 2]
```diff
@@ -64,5 +65,48 @@ def test_get_nodes_by_type(self):
         self.assertEqual(self.tree.get_nodes_by_type(3), [self.node2, self.node3])
 
 
+class TestSWCExport(unittest.TestCase):
+    def setUp(self):
+        self.input_file = "Case1_new.swc"
+        self.output_file = "Case1_exported.swc"
+
+    def test_load_export_compare(self):
+        # Load the original file
```
Comment (sanjayankur31) [inline tests/swc/test_LoadSWC.py:74]: add a one line docstring :)

**Review by sanjayankur31 — CHANGES_REQUESTED** [review]
Looks good, a few cosmetic tweaks to complete it.

**Comment by AdityaBITMESRA** [comment]
I have worked on the comments, please have a look if time permits.

**Comment by sanjayankur31** [comment]
Sounds good. Ill take a look on Monday.
-- 
Thanks,

Ankur

(Sent from mobile device)

**Comment by sanjayankur31** [comment]
Looks good. merging this in now. Do remember to pull to update your development and other branches.

**Pull request merged**